mkt/feed/tests/test_serializers.py

```diff
@@ -434,13 +446,15 @@ def setUp(self):
             else:
                 self.app_map[feed_element.app_id] = (
                     WebappIndexer.extract_document(feed_element.app_id))
-
-    def test_deserialize_many(self):
-        data = serializers.FeedItemESSerializer(self.data_es, context={
+        self.context = {
             'app_map': self.app_map,
             'feed_element_map': self.feed_element_map,
             'request': mkt.site.tests.req_factory_factory('')
-        }, many=True).data
+        }
+
+    def test_deserialize_many(self):
+        data = serializers.FeedItemESSerializer(
+            self.data_es, context=self.context, many=True).data
 
```

Comment:
Lots of test munging but it seems like mostly managing the context sent to the serializer.